package.json

```diff
@@ -8,8 +8,10 @@
     "node": "4.1.x"
   },
   "scripts": {
-    "start": "node index.js",
-    "test": "mocha test/*_test.js test/**/*_test.js"
+    "start": "node -r babel-core/register -r dotenv/config lib/index.js",
+    "prepublish": "babel lib -d dist",
```

Comment:
I also feel it's nicer to be explicit about babel and put it in the entry file. Less confusing that way...